ci-team.members.txt

```diff
@@ -10,3 +10,4 @@ yangpa@google.com
 gongyuan@google.com
 hnalla@redhat.com
 scottleehello@gmail.com
+dsun20@bloomberg.net
```

Comment:
Please alphabetize.

Please remove

* cartick
* clarktm
* kam.d.kasravi
* kunming
* lunkai

Reply:
@scottilee Are you still planning on contributing as part of the ci-team?

Reply:
removed the above list